lsb_vsx/patches/host_TETware/01_shared_mk.patch

```diff
@@ -5,7 +5,7 @@
  
  
 +# Ports are build with -j 9, below targets cannot be run in parallel due to mv command
```

Comment:
maybe update also the comment - as there are no "below targets" now?

Reply:
done